packages/mui-material/src/Radio/Radio.js

```diff
@@ -34,7 +34,11 @@ const RadioRoot = styled(SwitchBase, {
   overridesResolver: (props, styles) => {
     const { ownerState } = props;
 
-    return [styles.root, styles[`color${capitalize(ownerState.color)}`]];
+    return [
+      styles.root,
+      styles[`size${capitalize(ownerState.size)}`],
```

Comment:
We skip applying the `sizeMedium` class for the Radio component because `medium` is the default value, as can be seen in the `useUtilityClasses` above:
```suggestion
      ownerState.size !== 'medium' && styles[`size${capitalize(ownerState.size)}`],
```